runtime-modules/council/src/lib.rs

```diff
@@ -1184,41 +1184,32 @@ impl<T: Trait> ReferendumConnection<T> for Module<T> {
     fn can_unlock_vote_stake(vote: &CastVoteOf<T>) -> Result<(), Error<T>> {
         let current_voting_cycle_id = AnnouncementPeriodNr::get();
 
-        // allow release for very old votes
-        if current_voting_cycle_id > vote.cycle_id + 1 {
+        // If the vote is for an election prior to the last concluded..
```

Comment:
```suggestion
        // If the vote was cast before the latest Announcing stage...
```
The handbook should also be updated accordingly